ursaspace/ursa-arcgis-toolbox
# Feature/vcm 7571

### What's Changed:

- Analytic Descriptions
  - All analytic tool tip descriptions have been updated to provide more detail to users as well as any imagery restrictions
  - All change detection sub-options have been given tool tip descriptions as well 
  - Screen shots of the new descriptions have been added at the bottom of this description
- Updated Error Messaging
  - Error messages for Required AOI Field has been updated to provide more detail and information 

### To Test:
Analytic Descriptions: 
- To view/test that these have been updated hover over the "Information" icon that appears next to the "Analytic Options" and "Change Detection Options" selection pane. (Note for the Change Detection Options to show "Change Detection" must be selected in the "Analytic Options")

Updated AOI Error Message:
To Trigger the AOI Required Error Message do the following:
1. When filling out the “Analytics” form select the pencil to add a new AOI (select point for now). 
1.a This generates a feature layer to add points to
2. This will pre-populate the AOI field with a reference to the feature layer but no points have been added to the feature layer yet 
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/770549fc-9551-4910-a9d2-2a3fe40987b8)
2.a The contents pane will also show that the feature layer (Analytics Area of Interest (Points) 2) has a “Point” on it even tho we haven’t added on yet
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/bfe304ce-4e21-4075-a7f6-4ba8ae360c8f)
2.b This causes the analytic AOI field to display the error message since the feature layer is missing a point
 ![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/7161a61f-a8f2-4148-865c-47c7dfb892e5)
2.c Once a point has been added to the map the AOI field error will not clear until the Form is forced to refresh (either by selecting another analytic or updating a different field)

Analytic Desctiptions
Small Vessel Detection:
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/06ce5d6c-02e3-427d-a747-9948b0fbbbae)
Large Vessel Detection:
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/f471e7fd-1915-4a5d-bbc0-c6ba4a2e90d5)
Aircraft Detection:
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/25a1bf0d-234f-4d83-a5e9-6888631a0764)
Ground Vehicle Detection:
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/d7c39747-86c1-45a6-b995-3ed257dd427c)
Change Detection:
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/b559baf4-e0a5-4fcc-8e9c-215264602f73)
Ground Displacement:
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/8b0204f8-abdf-42c3-8398-de92656cff92)
Object Detection (SPEX):
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/bf038674-20cf-4c2f-8591-e1ad56d15e81)

Change Detection - Small Objects
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/a481ca75-5c03-48d3-9cfb-2943d87580a8)

Change Detection - Medium Structures & Large Areas
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/1f520d67-bdfb-4a37-a014-13382ff74853)
